docs/sources/_index.md

```diff
@@ -70,8 +70,9 @@ title: Grafana documentation
         <h4>Provisioning</h4>
         <p>Learn how to automate your Grafana configuration.</p>
     </a>
-    <a href="{{< relref "whatsnew/whats-new-in-v9-5/" >}}" class="nav-cards__item nav-cards__item--guide">
-        <h4>What's new in v9.5</h4>
+
+    <a href="{{< relref "whatsnew/whats-new-in-v10-0/" >}}" class="nav-cards__item nav-cards__item--guide">
+        <h4>What's new in v10</h4>
         <p>Explore the features and enhancements in the latest release.</p>
     </a>
```

Comment:
Welcome to Grafana 10! We are excited to introduce a series of powerful enhancements in areas including user experience, security, and plugin functionality. Read on to understand more about these significant changes in search and navigation, dashboards and visualizations, and authentication and security. For a more comprehensive dive into all the transformations in this release, please refer to the complete Grafana 10 documentation.

In this release, we've strongly emphasized creating a more intuitive user experience and delivering robust security measures. From bolstering our authentication protocols to debuting a host of new features like Grafana Correlations, Subfolders, Public Dashboards, Scenes, and new visualizations, Grafana 10 is designed to empower teams with the best tools for data visualization and monitoring. Stay tuned to discover how Grafana 10 can streamline your workflows, improve incident resolution, and enhance data security.